src/Sulu/Bundle/AdminBundle/Resources/js/stores/userStore/userStore.js

```diff
@@ -60,13 +61,25 @@ class UserStore {
             this.persistentSettings.set(key, persistentSettings[key]);
         });
 
-        // TODO this code should be adjusted/removed when a proper content-locale handling is implemented
-        // load and use first (default) localization of first webspace as content-locale for the user
-        localizationStore.loadLocalizations().then(action((localizations) => {
-            const defaultLocalizations = localizations.filter((localization) => localization.default);
-            const fallbackLocalization = defaultLocalizations.length ? defaultLocalizations[0] : localizations[0];
-            this.contentLocale = fallbackLocalization ? fallbackLocalization.locale : this.contentLocale;
-        }));
+        const contentLocale = this.persistentSettings.get(CONTENT_LOCALE_SETTING_KEY);
+
+        if (contentLocale) {
+            this.contentLocale = contentLocale;
```

Comment:
early return after this line instead of two `if`s?

Reply:
done